privaterelay/locales/en-us/app.ftl

```diff
@@ -0,0 +1,239 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## Brands
+
+# Dev Note: When adding to this section, use this file for naming conventions: https://github.com/mozilla/bedrock/blob/master/l10n/en/brands.ftl
+
+-brand-name-firefox = Firefox
+-brand-name-firefox-relay = Firefox Relay
+-brand-name-relay = Relay
+-brand-name-firefox-browser = Firefox Browser
+-brand-name-mozilla = Mozilla
+-brand-name-firefox-lockwise = Firefox Lockwise
+-brand-name-firefox-monitor = Firefox Monitor
+-brand-name-pocket = Pocket
+-brand-name-firefox-account = 
+    { $capitalization ->
+        *[lowercase] Firefox account
+        [uppercase] Firefox Account
+    }
+    
+# The file size limit for emails forwarded through the Relay service. KB is an abbreviation for kilobyte.  
+-email-size-limit = 150KB
```

Comment:
This doesn't really scale.

One issue is that terms should be used for things like brand/product names, where you might need private attributes. For this you can use a normal message (standard messages can be referenced in other messages, but also used stand-alone in code).

Second issue: what happens if you change the value? You need to update this string, which requires a new ID, and breaks all existing translations (which will require new IDs).

This should be

```
# Variables:
#   $size (number): maximum size for attachments
email-size-limit = { $size } KB
```

Even better, so we don't break if we switch to some MB.

```
# String used to display the attachment limit, e.g. 150 KB
# Variables:
#  $size (number): maximum size for attachments
#  $unit (string): unit of measurement (e.g. KB)
email-size-limit = { $size } { $unit }
unit-kb = KB
```

Questions on the English:
* Shouldn't there be a space between number and unit?
* `kB` or `KB`?

This should also move closer to the string that uses it.

Reply:
> Shouldn't there be a space between number and unit? 
Yes. Adding. 

> `kB` or `KB`?

KB because this file size limit is in kilobytes, rather than bits. (@groovecoder – Can you confirm?) 

---

I want to use the two-argument variable string. How do I reference `unit-kb`? In the example below I set the unit value in the `HTML`. 

**In the FTL:** 
```
faq-question-7-answer = We now support attachment forwarding. However, there is a { -email-size-limit } limit for email forwarding using { -brand-name-relay }. Any emails larger than { -email-size-limit } will not be forwarded.
```

**In the HTML:** 
```
<p class="faq-answer">{% ftlmsg 'faq-question-7-answer' %}</p>
```

```
<p class="faq-answer">{% ftlmsg 'faq-question-7-answer' size='150' unit='KB' %}</p>
```

Reply:
I passed arguments everywhere the string is referenced. I have the `unit-kb` variable defined in the FTL file, but not used anywhere. Please advise.

Reply:
> I want to use the two-argument variable string. How do I reference `unit-kb`? In the example below I set the unit value in the `HTML`.

I'm afraid I'm out of my depth when it comes to the actual code, and I'm not sure who else is familiar with this library at this point.

Worst case scenario, I would `KB` to `unit`, because having the message reference (i.e. `email-size-limit = { $size } { mb-unit }`) would have the same scalability/maintainability problem.

Reply:
Not sure if [this](https://django-ftl.readthedocs.io/en/latest/usage.html#lazy-translations) helps (can you store the unit-kb translation in variable, and pass that as the argument? I have no clue…)

Reply:
Okay – Talking with the team, this is okay as is. We are not planning on changing the limit any time in the near/far future. It is not on our roadmap.

Reply:
Not the first time we've been bitten by this approach…

Out of curiosity, the suggestion I linked to in the previous comment doesn't work? I'd still prefer to have that fully localizable and future-proof, if possible. If not, you should remove the "KB" string, since nothing is using it.

Reply:
Oh! Yeah – the previous comment didn't work because this is visible without any user interaction. The lazy function is for modals and things that only show when the user does an action to display them. However, this could be used for other strings in this application (Namely – all the strings used in `/includes/modal.html` and some in the `profile.html`). 

I'll file an issue to revist this. 

(also I'm removing that unused variable for now)